api/Element.json

```diff
@@ -223,6 +223,39 @@
           }
         }
       },
+      "anchorElement": {
```

Comment:
The spec PR adds this to `HTMLElement` not `Element`. Also, I don't see Chrome 125 shipping this. Should we remove this for now?

Reply:
Yeah this is not included in what's shipping so I wouldn't include it in this dataset.

Reply:
Ooops! In my next commit, I've moved it to `HTMLElement`.

The thing is though, it is supported, not behind a flag, in Canary (I've tested it). I'm guessing it just won't be shipped in Chrome stable for some time. So I feel like it is useful to keep this data for folks doing testing of preview features.

How can we represent features in this state? I thought we used to have some kind of "preview" value to use for this, but I can't find mention of it in the BCD data schema doc.

Reply:
Thanks for moving it. See https://github.com/mdn/browser-compat-data/pull/22813#pullrequestreview-2011180144 for setting preview and partial implementation because it is on the wrong interface at the moment.

Reply:
Just to confirm - the `anchor` attribute won’t ship in 125. (And current Canaries and Beta should not support it - Chris, please file a bug if you see some observable support.) We are still hoping to ship it soon once the spec discussion concludes, but I don’t know when that will be.

Reply:
@mfreed7 OK, filed at https://issues.chromium.org/issues/335744345. Is that OK? I've never filed a Chromium bug before ;-)

So I should just remove this for now?

And, can I clarify — this is just the `anchorElement` property we are talking about? The anchor` HTML attribute is still shipping?

Reply:
I'm removing the `anchor` attribute in https://github.com/mdn/browser-compat-data/pull/22910.

Reply:
Thanks for filing the bug - I just commented there. I suspect you have Experimental Web Platform Features enabled, which is why you see the `anchorElement` property available. Neither the `anchor` content attribute nor the `anchorElement` IDL attribute are shipping in Chrome in 125 or likely 126 at least.

Reply:
> @mfreed7 OK, filed at https://issues.chromium.org/issues/335744345. Is that OK? I've never filed a Chromium bug before ;-)

Also, yes of course, that was perfect!